app/views/contexts/thank_you_context.py

```diff
@@ -2,22 +2,47 @@
 from typing import Mapping
 
 from flask import url_for
+from flask_babel import format_datetime, lazy_gettext
 
 from app.data_models.session_data import SessionData
 from app.libs.utils import convert_tx_id
 from app.views.contexts.email_form_context import build_email_form_context
 
 
 def build_default_thank_you_context(
-    session_data: SessionData, submitted_at: datetime
+    session_data: SessionData, submitted_at: datetime, survey_type: str
 ) -> Mapping:
+    text = lazy_gettext("Your answers have been submitted for ")
```

Comment:
This should not be done using string concatenation.

Reply:
Removed concatenation, thanks!